pom.xml

```diff
@@ -985,6 +985,12 @@
                 <version>${hadoop.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-hdfs</artifactId>
```

Comment:
We don't need it, it's already covered by `hadoop-client-api` and `hadoop-client-runtime`

Reply:
Checked that hadoop-hdfs is included in org.apache.hadoop:hadoop-client-minicluster.
Removed hadoop-hdfs dependency here.